mobile/ios/Demo/Demo/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  EntangledDemo
+//
+//  Created by Rajiv Shah on 4/13/18.
```

Comment:
Remove authors from comments please

Reply:
Whoops, will do. I think Xcode automatically generates that